qds_sdk/commands.py

```diff
@@ -87,10 +87,26 @@ def run(cls, **kwargs):
         Returns:
             Command object
         """
+        err_pointer, tmp_pointer, new_bytes = 0, 0, 0
         cmd = cls.create(**kwargs)
         while not Command.is_done(cmd.status):
```

Comment:
@adeshr 
logs api work as told by client :). The reason it passes full logs is because we pass err_pointer as 0 and hence it passes entire log. This will happen here as well but we are handling this using logic inside:

> if err_length != "0":
new_bytes = int(err_length) + int(tmp_length) - new_bytes - tmp_pointer

Please tell if I am missing anything.